### ISSUE TITLE
installation issue

### ISSUE THREAD

**Issue by alessiaartusi** [Victorian-Bioinformatics-Consortium/prokka#634]
Hi everybody,

I am having an issue installing PROKKA, the error message is the following:
Can't locate Bio/Root/Version.pm in @INC (you may need to install the Bio::Root::Version module)

**Comment by reaset41** [comment]
I am having the same issue. Were you  able to resolve it?

**Comment by ambi1999** [comment]
Hi @jessicalumian ,

I am getting the same issue. Tried your method of 'conda install -c bioconda perl-bioperl ' but error persists. I am trying to install Prokka on 'Ubuntu 22.04 LTS'.

Can't locate Bio/Root/Version.pm in @INC (you may need to install the Bio::Root::Version module)

Thanks,
Ambi.

**Comment by ChooseelBunsuwansakul** [comment]
Hi @jessicalumian, it doesn't work.

**Comment by mmcguffi** [comment]
I also run into this issue -- I dont know much about Perl, but I also [encountered this problem using Roary and found a solution in their Github Issues](https://github.com/sanger-pathogens/Roary/issues/441#issuecomment-1088403325)

tldr; This solved it for me: 
`export PERL5LIB=$CONDA_PREFIX/lib/perl5/site_perl/5.22.0/`

My Perl version is v5.32.1 so Im not entirely sure why I have the directory to begin with, but it does solve the issue for me at least.

I believe this also relates to #446

**Comment by whipkreme** [comment]
Hi. can anyone help? im having this error while installing prokka.

>>
Collecting package metadata (current_repodata.json): done
Solving environment: unsuccessful initial attempt using frozen solve. Retrying with flexible solve.
Solving environment: unsuccessful attempt using repodata from current_repodata.json, retrying with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: unsuccessful initial attempt using frozen solve. Retrying with flexible solve.
Solving environment: / 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                                                                                                    

UnsatisfiableError:

**Comment by sterrettJD** [comment]
@whipkreme Are you trying to install it on a Mac with M1/M2 chip (ARM architecture)? I was having this issue on my Mac that was resolved when I used `CONDA_SUBDIR=osx-64 conda create -n prokka_env`, instead of `conda create -n prokka_env`.

See more info on conda environments with ARM architecture [here](https://towardsdatascience.com/how-to-manage-conda-environments-on-an-apple-silicon-m1-mac-1e29cb3bad12).
